Sources/Penny/Handlers/InteractionHandler.swift

```diff
@@ -27,11 +29,29 @@ struct InteractionHandler {
     private let oops = "Oopsie Woopsie... Something went wrong :("
     
     typealias InteractionOption = Interaction.ApplicationCommand.Option
+
     
     init(event: Interaction) {
         self.event = event
         self.logger[metadataKey: "event"] = "\(event)"
     }
+
+    private func makeJWTSigners() -> JWTSigner? {
+        let signer: JWTSigner
```

Comment:
I think this is left-over from the early states of this method? You can combine it with what is in line 51